src/components/shared/IdleSearchField.js

```diff
@@ -122,12 +117,13 @@ class IdleSearchField extends PureComponent<Props, State> {
           onChange={this._onSearchFieldChange}
           onFocus={this._onSearchFieldFocus}
           onBlur={this._onSearchFieldBlur}
+          ref={this._takeInputRef}
         />
         <input
           type="reset"
           className="idleSearchFieldButton"
           onClick={this._onClearButtonClick}
-          onFocus={this._onClearButtonFocus}
+          tabIndex={-1}
```

Comment:
Thinking about it more, I don't think we want to avoid to focus it using the keyboard, so we can remove this tabIndex attribute IMO. The previous behavior was really done for the mouse. What do you think?